packages/nimble-components/src/utilities/unit-format/decimal-unit-format.ts

```diff
@@ -53,6 +54,37 @@ export class DecimalUnitFormat extends UnitFormat<DecimalUnitFormatOptions> {
         this.maximumFractionDigits = maximumFractionDigits;
     }
 
+    public static normalizeAndDefaultFractionDigitOptions(
```

Comment:
Putting `normalizeAndDefaultFractionDigitOptions` in the `DecimalUnitFormat` doesn't seem quite right to me because these defaults are something specific to our choices for the number-text column and our pipe, but not something that is fundamental to the `DecimalUnitFormat` class itself.

I'm not quite sure what a better location is, other than creating a new file to encapsulate this logic. Did you consider any location alternatives, or have a strong rationale of why it makes sense here?

Reply:
I didn't like the idea of creating a new file for this one function, especially because it is so closely related to our `decimal` number formatting mode. `DecimalUnitFormat` models that mode, so I put the function there.

One possibility is to change `DecimalUnitFormat` so that it takes `decimalDigits` instead of `minimumFractionDigits`, and move the normalization/default logic into the constructor. The only reason I hesitate to do that is that the current API is more flexible and closer to the `Intl.NumberFormat` API. Another option is just to _add_ the `decimalDigits` parameter rather than replacing `minimumFractionDigits`. Either of these options would result in a messier API (i.e. exposing incompatible parameters), but the latter would at least preserve flexibility and compatibility with the `Intl.NumberFormat` API. Do you like that better than what I have currently? Or would you argue for moving the function to a new file?

Reply:
I agree that keeping the `DecimalUnitFormatter` accepting `minimumFractionDigits` and `maximumFractionDigits` seems the best.

Part of what I'm considering is that I thought one goal was for the code within `src/utilities/unit-format` to represent what could be a third-party library (or split into a different package in the future). So, I guess my questions are:
- Is that a goal?
- If it is a goal, do we think that the defaults we've chosen for the number column belong in that same directory?

Reply:
That's fair. If we truly want to `src/utilities/unit-format` to be standalone, it doesn't make much sense to add a secondary API with different defaults, just because that's what we want for the pipe and number column. I'll separate it.